tracer/src/Datadog.Trace/Configuration/ConfigurationSources/Telemetry/ConfigurationBuilder.cs

```diff
@@ -59,41 +25,13 @@ public ConfigurationBuilder(IConfigurationSource source, IConfigurationTelemetry
 
     public HasKeys WithKeys(string key, string fallbackKey1, string fallbackKey2, string fallbackKey3) => new(_source, _telemetry, key, fallbackKey1, fallbackKey2, fallbackKey3);
 
-    private static bool TryHandleResult<T>(
```

Comment:
This was inlined as it was only ultimately used in 1 place, and is much more complicated to understand than is worth it